tests/builtin/secrets_test.go

```diff
@@ -24,6 +24,6 @@ func Test_Database_Secret_DefaultParameters(t *testing.T) {
 	expected := "t-silent-test-1234/c-green-test-1234/builtin/db-password"
 	assert.Len(t, data, 4)
 	assert.Equal(t, expected, data["KC_DB_PASSWORD"])
-	assert.Equal(t, expected, data["postgresql-password"])
-	assert.Equal(t, expected, data["postgresql-postgres-password"])
+	assert.Equal(t, expected, data["postgres-password"])
```

Comment:
what's the benefit to rename it from postgre*sql*-password to postgres-password? 

in [component/main.jsonnet](https://github.com/projectsyn/component-keycloak/pull/147/files?diff=unified&w=0#diff-f32a1b7e527d62d60748c4d0db809c92a78ab55ded0c63ef19f77cbf12f2c568) we have another set of naming:
```
   'postgresql-postgres-password': params.database.password,
    'postgres-password': params.database.password,
``` 
I suppose that's intentional and not a typo.

Reply:
Yes, this is tested against Keycloak and Postgres.